vignettes/baselinenowcast.Rmd

```diff
@@ -270,9 +274,9 @@ The `get_delay_estimate()` function expects the following inputs:
   - `max_delay`: an integer indicating the maximum delay to estimate. This must
   be less than or equal to the number of rows in `triangle` minus 1, since we
   assume `triangle` is indexed at 0.
-  - `n_history_delay`: an integer indicating the number of observations by reference
+  - `n`: an integer indicating the number of observations by reference
   date to use to fit the delay distribution. This must be less than or equal to
-  the number of rows in `triangle`.
+  the number of rows in `triangle`. Here we will use `n_history_delay`.
```

Comment:
mention somewhere we could have done the delay and applied it for a point nowcast in one step to better link to the list iterator coming in the next section.

Reply:
Added a sentence